File: src/BukkitPE/metadata/PlayerMetadataStore.php
<?php
namespace BukkitPE\metadata;

use BukkitPE\IPlayer;

class PlayerMetadataStore extends MetadataStore{

	public function disambiguate(Metadatable $player, $metadataKey){
		if(!($player instanceof IPlayer)){
			throw new \InvalidArgumentException("Argument must be an IPlayer instance");
		}

		return strtolower($player->getName()) . ":" . $metadataKey;
	}
}

File: src/main/java/net/BukkitPE/metadata/PlayerMetadataStore.java
package net.BukkitPE.metadata;

import net.BukkitPE.IPlayer;

/**
 * BukkitPE Project
 */
public class PlayerMetadataStore extends MetadataStore {

    @Override
    protected String disambiguate(Metadatable player, String metadataKey) {
        if (!(player instanceof IPlayer)) {
            throw new IllegalArgumentException("Argument must be an IPlayer instance");
        }
        return (((IPlayer) player).getName() + ":" + metadataKey).toLowerCase();
    }
}
